webapp/lib/app/configurator/tags/view.dart

```diff
@@ -55,76 +56,74 @@ class TagsConfigurationPageView extends ConfigurationPageView {
   }
 }
 
-class TagGroupView {
-  DivElement _tagsGroupElement;
-  DivElement _tagsContainer;
-  DivElement _tagsHeader;
+class TagGroupView extends AccordionItem {
   String _groupName;
   TextEdit editableTitle;
-  Button _addTagButton;
+  DivElement _header;
+  DivElement _body;
+  DivElement _tagsContainer;
+  Button _addButton;
 
   Map<String, TagView> tagViewsById;
 
-  TagGroupView(this._groupName) {
+  TagGroupView(id, this._groupName, this._header, this._body) : super(id, _header, _body, false) {
     _groupName = _groupName ?? '';
-    _tagsGroupElement = new DivElement()..classes.add('tags-group');
-    var tagsDropzone = new dnd.Dropzone(_tagsGroupElement);
+
+    editableTitle = TextEdit(_groupName, removable: true)
+      ..onEdit = (value) {
+        _view.appController.command(TagsConfigAction.updateTagGroup, new TagGroupData(_groupName, newGroupName: value));
+        _groupName = value;
+      }
+      ..onDelete = () {
+        requestToDelete();
+      };
+
+    _header.append(editableTitle.renderElement);
+
+    _tagsContainer = DivElement()..classes.add('tags-group__tags');
+    _body.append(_tagsContainer);
+
+    _addButton = Button(ButtonType.add);
+    _addButton.renderElement.onClick.listen((e) {
+      _view.appController.command(TagsConfigAction.addTag, new TagData(null, groupId: _groupName));
+    });
+    _body.append(_addButton.renderElement);
+
+    var tagsDropzone = new dnd.Dropzone(renderElement);
+    tagsDropzone.onDragOver.listen((event) {
+      renderElement.classes.toggle('dropzone--active', true);
+    });
+    tagsDropzone.onDragLeave.listen((event) {
+      renderElement.classes.toggle('dropzone--active', false);
+    });
     tagsDropzone.onDrop.listen((event) {
       var tag = event.draggableElement;
       var tagId = tag.dataset['id'];
       var groupId = tag.dataset['group-id'];
+      expand();
```

Comment:
can we change the logic for expanding the accordion to be done on drag over rather than on drop? so:
* onDragOver: if not expanded, expand and remember that it was collapsed
* onDragLeave: if it was collapsed before drag, collapse it again

Reply:
Oh I had originally tried this but it is quite janky
If there are say 10 groups A, B, C ... with everything collapsed at the start
Say we drag a tag from A to F

Once out of A, B opens. Once out of B, B is closed but the hover position could be such that it is over E that the group E opens. All this happens quite fast that sometimes it is confusing as to what group you are hovering over. 

Please let me know if there is a better way to do this?

Reply:
hmm, yeah, I can see how that would happen. let's keep the current behaviour as is, and look to implement the target behaviour later on, as I think it'll be quite complex (e.g. adjusting the page scroll when folding so that the page appears stable relative to the cursor position on the page). I've opened an issue for tracking this: https://github.com/larksystems/Katikati-Core/issues/548